### PR TITLE
fixed dashboard's background height

### PR DESCRIPTION
**Pull request by biswajit035**
## Describe your changes
I have fixed **DASHBOARD'S BACKGROUND**

## Screenshots - If Any (Optional)
**BOFORE**
image

***AFTER**
image

## Checklist before requesting a review

- [x] I have performed a self-review of my code.

- [x] Followed the repository's [Contributing Guidelines](https://github.com/dvstechlabs/Noteslify/blob/main/CONTRIBUTING.md).

- [x] I ran the app and tested it locally to verify that it works as expected.

- [x] I have starred the repository.

### CODE REVIEW

File: frontend/src/App.css [hunk 1]
```diff
@@ -87,7 +87,7 @@ body {
 	top: 0;
 	top: 0;
 	left: 250px;
-	height: 100vh;
+	min-height: 100vh;
```
Comment (biswajit035) [inline frontend/src/App.css:90]: changed for bigger page size

File: frontend/src/components/Sidenav/Sidenav.css [hunk 2]
```diff
@@ -214,6 +214,6 @@ body.dark .sidebar li a:hover .text {
 
 .sidebar.close~.home {
     left: 78px;
-    height: 100vh;
+    min-height: 100vh;
```
Comment (biswajit035) [inline frontend/src/components/Sidenav/Sidenav.css:217]: changed for bigger page size

**Review by biswajit035 — COMMENTED** [review]
done

**Review by MrKrishnaAgarwal — APPROVED** [review]
Looks good to me!
Thanks for contributing 🎉

File: frontend/src/App.css (hunk 1 above)
Comment (MrKrishnaAgarwal) [reply frontend/src/App.css:90]: Mark this as resolved ✔️ @biswajit035

**Review by devarshishimpi — APPROVED** [review]
Thanks For Contributing!!!

**Pull request merged**